api.md

```diff
@@ -0,0 +1,86 @@
+# Proposed webrtc webextension api
+
+The initial version of the webrtc webextension api is a limited
+set of events, just enough to implement the extension from
+[this mozhacks article](https://hacks.mozilla.org/2015/09/controlling-webrtc-peerconnections-with-an-extension/).
+
+To use any of these events, an extension must have the `"webrtc"`
+permission (in addition to the `"experiments.webrtc"` permission unless/until
+this api lands in central).  In addition, an extension must have the
+`"webrtc.blocking"` permission to add a blocking onPeerConnection listener.
+
+The events are:
+
+## onPeerConnection
```

Comment:
Those names work for me. Permission is the primary reason for these hooks anyhow, and that's affected their design. E.g. they're invoked as a gate per RTCPeerConnection instance, i.e. once an instance has been "granted permission" you won't see any subsequent requests from that object. Conversely, a page may have several RTCPeerConnection objects, and it's up to an extension to avoid bombarding a user with more prompts than necessary.